doc/man3/OSSL_ENCODER_CTX.pod

```diff
@@ -4,10 +4,24 @@
 
```

Comment:
Would I be correct in saying that this pod file is not particularly useful to the normal average user of the library.
It looks like it would be useful for provider writers that need to implement their own encoders.
Maybe it should state something along those lines in the description, as this is not particularly light reading.
The same comment applies to the decoder also.

Reply:
I would rather say that this is meant to be useful for those who want to write a new `OSSL_ENCODER_CTX_new_by_TYPE()`
That's not necessarily provider authors.

Reply:
Something short, that may stop someone from trying to understand this pod file if they dont have to -  is fine..

Reply:
... and what other pages?

Reply:
That's not me being snappy, btw.  We have other pages with similar levels of complexity and no such warning, so starting somewhere could end up being bigger than this PR.

Reply:
ok - up to you..

Reply:
My suggestion is that we make a collective mental note of this.
(on my mental backlog of things to do is a lot of documentation cleanup during the beta period...  among others, we have a mixture of styles, with an increasing number of pages following the man-pages(7) recommendations, and the remaining lot that have the [ahem] home grown style...  I plan on clearing that up)